perun-cli/Perun/auth/OidcAuth.pm

```diff
@@ -386,13 +384,20 @@ sub revokeToken
 sub checkConfigExists
 {
 	my $configName = $_[0];
-	if ($configName eq $UNDEFINED_CONFIGURATION) {
-		die "ERROR: environment variable PERUN_OIDC_CONFIG is not set!\n";
-	}
 	my %configs = % { loadAllConfigurations() };
+
 	if (!exists $configs{$configName}) {
 		die "ERROR: configuration \"$configName\" does not exist!\n";
 	}
 }
 
+sub getDefaultConfig
+{
+	my %configs = % { loadAllConfigurations() };
+	if (length keys %configs eq 0) {
+		die "ERROR: configuration file is empty!\n";
```

Comment:
This for some reason does not get thrown, but compilation error arises.

Reply:
A compilation error would occur when converting an empty file to hash, so used try block instead.